docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+
+!!! example "Explore the Material for MkDocs Blog"
+
+    Take a moment to look at the [Material for MkDocs blog] to see what elements
+    a blog consists of before moving on to the next section. What makes the blog
+    different from the rest of the site?
+
+[Material for MkDocs blog]: https://squidfunk.github.io/mkdocs-material/blog/
+
+## 3. Key concepts
+
+A blog consists of a number of self-contained _posts_ (also often called
+articles).
+
+An index page shows the posts in reverse chronological order, with the most
+recent post at the top. It usually shows only a short _excerpt_ and a link that
+the user can click to navigate to the full post.
+
+Both the index page and the post itself usually list information such as
+when you published the post, when you updated it, who the author is and what the
+expected reading time is.
+
+Since the blog posts are primarily arranged by time and not into a hierarchy,
+their URLs do not reflect such a structure. Instead, each post's URL usually
+contains a shortened description, the _slug_, which is usually derived from
+the first heading in the post.
+
+The main navigation structure is the timeline, which you can subdivide into
+_categories_. In addition, posts can be _tagged_ to provide an additional
+navigation structure based on content.
+
+The navigation elements in the blog are the timeline, with the main index page
+showing a given number of posts and an _archive_ section allowing access to
+older posts, organized by year. The _categories_ section provides access to
+index pages for the categories. In addition, when using tagging, you can also
+create tag index pages.
+
+Navigation by _author_ helps users to find posts by a specific user when there
+are more than one. Each author index page lists posts by that author in reverse
+chronological order and can provide more information about them.
+
+Finally, an _RSS feed_ allows users to subscribe to a blog so that they get
+notified when you publish new posts. RSS Feed readers are often used to access
+blogs that a user follows. They usually support downloading the blog content for
+offline consumption.
+
+## 4. Setting up your blog
+
+The blog plugin is part of Material for MkDocs but you need to configure it
+in the `mkdocs.yml`.
+
+!!! example "Set up a blog"
+
+    If you have not done so already, create a project for your blog:
+
+    === "MacOS/Linux"
+        ```bash
+        $ mkdocs new myblog
+        $ cd myblog
+        ```
+
+    === "Windows"
+        TODO
+
+    Then edit the `mkdocs.yml` file in the newly created directory to make sure
+    if has the following content:
+
+    ```yaml
+    site_name: Blog Tutorial
+    site_description: an example blog set up following the tutorial
+    site_url: http://www.example.com
+
+    theme:
+      name: material
+
+    plugins:
+      - search
+      - blog
+    ```
+
+The blog plugin will create a directory structure for your blog posts if it does
```

Comment:
Why not let the user everything do when running `mkdocs serve`? It's a much better experience than running `mkdocs build` and inspecting the `site` directory, IMHO. The blog plugin should be resilient enough and if not, we should ensure that it is.